src/styles/components/side-panel-detail.less

```diff
@@ -226,6 +226,23 @@ Side Panel Detail
 
     width: @base-spacing-unit * 7;
 
+    &-wide {
+
+      width: @base-spacing-unit * 12;
```

Comment:
I know, but the margin should still change along with everything else when screen becomes smaller or bigger.
The `base-spacing-unit` has different values for different screens